### DISCUSSION TITLE
Binder Design Size

### DISCUSSION THREAD

**Discussion by JA-Hull** [RosettaCommons/RFdiffusion#56] [General]
There's a good amount of detail about RF diffusion followed by ProteinMPNN-FastRelax in terms of selecting target sites, truncating proteins, scale, and filtering. However, there's not really any discussion about how long to design the binder. I have reasonably good in silico success with 25 and 30mer peptides (getting interaction PAE ~5) when I have really small protein pockets. Is there a general guidance on how long these proteins should be? Is it just guess and check or are there suggested minimum/maximums?

**Comment by kjogr12** [comment]
I'm also interested in binder design for peptide/small proteins(< 20 AA).
Is there a size limitation for proteins with this method? Or could you inform me about the protein size range that has been validated with this approach?

**Reply by xinran1129305** [reply]
Hi there. 
I'm wondering that if I want to design small binders, do I need to set and try different symmetry settings? Almost all of my designs failed, since they cannot meet the criteria of in silico success:( 
Thanks in advance!
